### DISCUSSION TITLE
Scroll support on Opensearch Serverless

### DISCUSSION THREAD

**Discussion by cscetbon** [aws/aws-sdk-pandas#2639] [Q&A] [answered]
I see in the code that if scroll is used with OpenSearch Serverless then an exception is raised https://github.com/aws/aws-sdk-pandas/blob/3.5.1/awswrangler/opensearch/_read.py#L109-L110. However I can't find that in the official documentation. Is it still true ?

**Comment by kukushking** [comment, marked as answer]
Hi @cscetbon , unfortunately `GET _search/scroll` is [not](https://docs.aws.amazon.com/opensearch-service/latest/developerguide/serverless-genref.html) in the list of supported operations.

I just ran a quick test to double check and can confirm the above - it is not yet supported.
